elastic-container.sh

```diff
@@ -18,8 +18,10 @@
 ELASTIC_USERNAME="elastic"
 ELASTIC_PASSWORD="password"
 ELASTICSEARCH_URL="http://elasticsearch:9200"
+LOCAL_ES_URL="http://127.0.0.1:9200"
 KIBANA_URL="http://kibana:5601"
 LOCAL_KBN_URL="http://127.0.0.1:5601"
+FLEET_URL="http://fleet-server:8220"
 STACK_VERSION="7.16.3"
```

Comment:
```suggestion
STACK_VERSION="7.17.0"
```

Reply:
I tested this and it works.

![image](https://user-images.githubusercontent.com/7442091/152080205-4564f087-45e4-4dbf-a832-1bf8e41dd436.png)